client/ayon_kitsu/hooks/pre_status_cahnge.py

```diff
@@ -0,0 +1,31 @@
+from ayon_applications import PreLaunchHook
+import os
+
+class pre_status_change(PreLaunchHook):
+    order = 1
+    launch_types = ()
+    note_status_shortname="wip"
+
+    def execute(self):
+        import gazu
+        kitsuId= self.data["task_entity"]['data']['kitsuId']
+        
+        self.log.debug(f'Changing Kitsu task status to WIP. Task id {kitsuId}')
+
+        try:
+            gazu.set_host(os.environ["KITSU_SERVER"])
+
+            gazu.log_in(os.environ["KITSU_LOGIN"], os.environ["KITSU_PWD"])
+
+            kitsu_status = gazu.task.get_task_status_by_short_name(
+                        self.note_status_shortname
+                    )
+
+
+            task=gazu.task.get_task(kitsuId)
+
+            gazu.task.add_comment(task["id"], kitsu_status)    
+
+            gazu.log_out() 
+        except:
```

Comment:
It's not recommended to use bare except. 
Could we be more explicit about the error type ?

Reply:
I have this message 

Message: 'Expected keys from `get_plugin_paths` are "publish", "create", "load", "actions", "inventory". Addon: "kitsu" - got key "hooks"'

Reply:
If that is the reason then just remove `"hooks"` from `get_plugin_paths` and you don't need to capture anything.